app/inject/TemplateServiceProvider.java

```diff
@@ -35,7 +35,7 @@ public TemplateService get() {
                     } else if ("file".equals(type)) {
                         return new FileTemplateLoader(path);
                     } else {
-                        throw new RuntimeException("cannot build template loader for " + loaderConfig);
+                        throw new RuntimeException("cannot create template loader for " + loaderConfig);
```

Comment:
I created SunriseInitializationException for problems on fetching configuration and data needed to start the application, so I think this can be a candidate to use it.